2-Draft-Accepted/RFCNNNN-Native-Markdown-Rendering.md

```diff
@@ -138,4 +174,5 @@ The limitations of this approach are:
 
 ## Open Questions
 
-- Investigate line wrapping behavior.
\ No newline at end of file
+- Investigate line wrapping behavior.
```

Comment:
As currently we discussed (@SteveL-MSFT ) tabs is problematic - we should ignore a console settings and do rendering in PowerShell consolehost internally.

Reply:
I think "line wrapping" here refers to semantic line breaks?

Reply:
After some investigation, we do not have to special case for this. This is parsed automatically.